### PR TITLE
Create devops docs

### PR DESCRIPTION
**Pull request by jschmidt-civicactions**
Describe next-build deploy process

## Description
Documentation update.
Relates to #[17706](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17706). 

## Testing done
n/a

## Screenshots
n/a

## QA steps
n/a

### CODE REVIEW

File: READMEs/devops/deploy.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Deploy Process
```
Comment (tjheffner) [inline READMEs/devops/deploy.md:1]: How do you feel about changing this readme to `EKS.md` and this line to `EKS Deploy Process`? I just feel like `deploy` is a loaded term that could apply to a number of different things (EKS, deploying released content, etc)

File: READMEs/devops/deploy.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Environment updates run through the following steps using GitHub Actions and ArgoCD:
+
+1. Run ["Create Production Tag" workflow](https://github.com/department-of-veterans-affairs/next-build/actions/workflows/production-tag.yml)
```
Comment (tjheffner) [inline READMEs/devops/deploy.md:5]:
```suggestion
1. Run ["Create Production Tag" workflow](https://github.com/department-of-veterans-affairs/next-build/actions/workflows/production-tag.yml)
- This happens automatically after a commit is added to the `main` branch and passes the CI workflow.
```

File: READMEs/devops/deploy.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Deploy Process
+
+Environment updates run through the following steps using GitHub Actions and ArgoCD:
```
Comment (tjheffner) [inline READMEs/devops/deploy.md:3]:
```suggestion
Environment updates run through the following steps using GitHub Actions and ArgoCD. These happen automatically once a commit is added to the `main` branch and passes the CI workflow.
```

Comment (jschmidt-civicactions) [reply READMEs/devops/deploy.md]: Doesn't this only run with a new production tag, not any passing commit to main?

Comment (tjheffner) [reply READMEs/devops/deploy.md]: a passing commit to main triggers the new production tag, which triggers the rest of it

Comment (jschmidt-civicactions) [reply READMEs/devops/deploy.md]: Aha! I missed that bit in the workflow yaml.

**Review by tjheffner — APPROVED** [review]

**Pull request merged**